src/cs50/sql.py

```diff
@@ -211,3 +223,16 @@ def process(value):
         else:
             self.logger.debug(termcolor.colored(log, "green"))
             return ret
+
+
+# http://docs.sqlalchemy.org/en/latest/dialects/sqlite.html#foreign-key-support
+def _on_connect(dbapi_connection, connection_record):
```

Comment:
Is `_on_connect` conventional? Or should we just call it `_connect` so that it matches the event name?